src/Restaurant/index.js

```diff
@@ -1,11 +1,15 @@
 import './styles.scss';
-const Restaurant = ({ restaurantData }) => {
+
+const Restaurant = ({ restaurantData, heroHeader, setRestaurantId}) => {
```

Comment:
remove additional spaces inside()

also heroHeader doesn't look like its used in this component as its greyed out in PHP storm - think this can be removed?